src/dynamics/mpas/dyn_grid.F90

```diff
@@ -185,6 +168,18 @@ subroutine dyn_grid_init()
 
    ! Query global grid dimensions from MPAS
    call cam_mpas_get_global_dims(nCells_g, nEdges_g, nVertices_g, maxEdges, nVertLevels, maxNCells)
+
+   ! Temporary global arrays needed by phys_grid_init
+   allocate(lonCell_g(nCells_g))
+   allocate(latCell_g(nCells_g))
+   allocate(areaCell_g(nCells_g))
+   call cam_mpas_get_global_coords(latCell_g, lonCell_g, areaCell_g)
+   
```

Comment:
@MiCurry @PeterHjortLauritzen I'm thinking this code is an accidental merge as I deleted lonCell_g, latCell_g, and areaCell_g as part of weakscaling updates.  Just making sure that neither of you put this back in to cam_mpas_dev_pel for testing or debug or something.  This doesn't compile currently.

Reply:
Thanks. Fixed